### ISSUE TITLE
Not finding 4kb deletion

### ISSUE THREAD

**Issue by ccbruels** [cytham/nanovar#75]
Hi,
I ran NanoVar on a ~20x coverage ONT data set. I noticed that it is not calling a ~4kb deletion (screenshot attached). I can't find a max SV length parameter, is there something I can do to call SV like this?
BTW - the reports are very helpful!
Thanks!
Christine
![NanovarNotFindingDeletion](https://github.com/cytham/nanovar/assets/36895206/605291f2-98c7-451d-820e-e5d832f6449d)

**Comment by cytham** [comment]
Hi @ccbruels, is this hg38? Can you share the genomic coordinate of the deletion?

**Comment by ccbruels** [comment]
Hi ChengYong,

It is hg38, the deletion is chr10:131592486-131597187. The deletion was
called by Sniffles1, Sniffles2, and DeBreak. I noticed the deletion because
IGV gave an error message when I tried to load the vcfs from DeBreak and
Sniffles2. It can load the vcf with the deletion from Sniffles1, and the
vcf from NanoVar which didn't call the deletion.

When I looked at it more closely after writing you, I see that NanoVar is
the only one of the 4 callers that called it correctly (in this case NOT
calling an SV).

The sequence that was reported as being deleted is composed of a SINE
(AluSg) and 2 LINES (L1MB3 and L1PREC2). The initial assembly of hg38 has
this same SINE and 2 LINES followed by a very similar
AluSg-L1MB3-L1PREC2/L1PA14. There is now a fix (chr10_KN538365v1_fix) which
does not include that first AluSg/L1MB3/LiPREC2. This fits with our ONT
data which indicates that the first AluSg/L1MB3/LiPREC2 in hg38 is likely
not present.

Best,
Christine
